### PR TITLE
Have Travis check the build for Dart errors.

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
@abarth this is ready now

File: travis/setup.sh [hunk 1]
```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 set -ex
 
-./tools/dart/update.py
+export CLOUDSDK_CORE_DISABLE_PROMPTS=1
+curl https://sdk.cloud.google.com | bash
```
Comment (abarth) [inline travis/setup.sh]: You probably dont need this.

**Comment by abarth** [comment]
LGTM

File: travis/setup.sh (hunk 1 above)
Comment (Hixie) [inline travis/setup.sh]: oh right, I added that when trying to figure out what "download_from_google_storage" was, but that turns out to be in depot_tools. Let me try removing it before I check it in...

**Pull request merged**